src/main/java/com/useful/ucars/uCarsListener.java

```diff
@@ -1275,20 +1293,13 @@ public void run() {
 				theNewLoc.add(0, 1.5d, 0);
 				Boolean calculated = false;
 				double y = 1.1;
-				if (block.getType().name().toLowerCase().contains("step")) {
+				if (block.getType().name().toLowerCase().contains("slab")) {
 					calculated = true;
 					y = 1.2;
 				}
-				if (carBlock.name().toLowerCase().contains("step")) { // In
-																		// a
-																		// step
-																		// block
-																		// and
-																		// trying
-																		// to
-																		// jump
+				if (carBlock.name().toLowerCase().contains("slab")) { // In a slab block
 					calculated = true;
-					y = 0.6;
+					y = 1.5;
```

Comment:
Why 1.5 here?

Reply:
Will double check but I believe one was stuck on that slab otherwise